gatsby-config.js

```diff
@@ -42,6 +44,51 @@ module.exports = {
         },
       },
     },
+    {
+      resolve: 'gatsby-source-swiftype',
+      options: {
+        file: `${__dirname}/src/data/related-pages.json`,
+        refetch: Boolean(process.env.BUILD_RELATED_CONTENT),
```

Comment:
Is this for caching reasons?

Reply:
Yes. By default, the data is resolved from the `src/data/related-pages.json` file. When we need to refresh that data and make new requests to Swiftype, setting the `BUILD_RELATED_CONTENT` environment variable will instruct the plugin to go make new requests to Swiftype and save them in the JSON file.